### ISSUE TITLE
Add page for Imprint, Privacy Policy, Accessibility

### ISSUE THREAD

**Issue by rombaum** [ts4nfdi/service-portal#27]
We should add information pages for Imprint, Privacy Policy and Accessibility. On this page should be a sentence like the one from [DANTE](https://dante.gbv.de/search) (see figure). We didn't need an overlay/popup for this like Dante is showing it. If this would be easier to implement go on.

This page was a suggestion by the Base4NFDI service steward for legal aspects. Here are some suggestions.

**_For Accessibility_**
```
The corresponding accessibility statement is located on a page outside the TS4NFDI Service Portal web app and is therefore opened in a new tab.

Go to the accessibility explanation at https://base4nfdi.de/accessibility
```

**_Privacy Policy_**
```
The corresponding privacy policy is located on a page outside the TS4NFDI Service Portal web app and is therefore opened in a new tab.

Go to the privacy policy at https://www.zbmed.de/en/privacy-policy
```

**_Imprint_**
```
The corresponding legal notice is located on a page outside the TS4NFDI Service Portal web app and is therefore opened in a new tab.

Go to the imprint at https://www.zbmed.de/en/legal-notice
```

example DANTE
![Image](https://github.com/user-attachments/assets/e06f101d-126a-45b1-a5f9-f1049600fab3)

**Comment by nichtich** [comment]
For other applications we have three links in the header (e.g. [Cocoda](https://coli-conc.gbv.de/cocoda/app/)) or at the footer (e.g. [BK](https://uri.gbv.de/terminology/bk/)). Maybe footer links are enough.

**Comment by rombaum** [comment]
We did the same in the past but we got an advise from Base4NFDI that we should say that the user will be directed to other web page.
